mldsa/src/sign.c

```diff
@@ -230,6 +230,7 @@ int crypto_sign_keypair_internal(uint8_t pk[CRYPTO_PUBLICKEYBYTES],
   return 0;
 }
 
+#if !defined(MLD_CONFIG_NO_RANDOMIZED_API)
```

Comment:
Shouldn't we modify `sign.h` as well?

I'm not sure we can/should do something about `mldsa_native.h`. We don't have access to the config in there. In mlkem-native, I think it's a (harmless) bug that `mlkem_native.h` uses `MLK_CONFIG_NO_RANDOMIZED_API`.

Reply:
Ah... mlkem-native also does not alter `kem.h`. We should leave it or change it in both places.